### PR TITLE
Test

### PR DESCRIPTION
**Pull request by bialesdaniel**
Attempting to deploy things to staging envrionment

### CODE REVIEW

**Comment by bialesdaniel** [comment]
> actually, after reviewing the port usage, i somehow prefer incremental port usage from `30000`, like
> 
> * pss: `30001`
> * grafana: `30002`
> * timescaledb: `30003`
> 
> otherwise, we might easily lose track of what ports have been used and what are available

I originally tried to do this but I got an error saying port 30001 was in use.

File: performance-storage-service/hello.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+klasdflkasdf
```
Comment (skyfishjy) [inline performance-storage-service/hello.txt:1]: Delete this file

File: grafana/trigger.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+1
+2
+3
```
Comment (skyfishjy) [inline grafana/trigger.txt:3]: Delete this file

File: timescaledb/trigger.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+1234567891098765432101
```
Comment (skyfishjy) [inline timescaledb/trigger.txt:1]: Delete this file

**Pull request merged**